recipes/libcurl/all/conanfile.py

```diff
@@ -554,6 +556,8 @@ def _configure_cmake(self):
             self._cmake.definitions["USE_LIBIDN2"] = self.options.with_libidn
         self._cmake.definitions["CURL_DISABLE_RTSP"] = not self.options.with_rtsp
         self._cmake.definitions["CURL_DISABLE_CRYPTO_AUTH"] = not self.options.with_crypto_auth
+        if tools.Version(self.version) >= "7.83.1":
+            self._cmake.definitions["CURL_DISABLE_VERBOSE_STRINGS"] = not self.options.with_verbose_strings
```

Comment:
```suggestion
        self._cmake.definitions["CURL_DISABLE_VERBOSE_STRINGS"] = not self.options.with_verbose_strings
```
Doing a quick check, that option is available since 7.64.1 (oldest supported version in Conan Center), which means, you don't need to exclude old versions.

Reply:
great! was being cautious by adding the version check